### ISSUE TITLE
On classification result

### ISSUE THREAD

**Issue by OolongQian** [princeton-vl/SimpleView#2]
Congrats on the paper. Your work is very insightful.

I have downloaded the code and trained the classification with config file dgcnn_{model_name}_run_1.yaml. 

I test the model performance using 'best test model selection' protocol. During test pointnet++ gets 92.95% classification accuracy, pointnet gets 88.04%, simpeview gets 92.54%. 
However, in paper Table 3, using DGCNN protocol with smoothed cross-entropy loss, I believe pointnet++ should get 93.1%, pointnet 90.5%, and for simpeview is 93.6%. 
I would like to ask how to achieve the reported performance reliably? Do I need to run multiple times with different random seeds and do some extra processing? 

Thanks in advance.

**Comment by imankgoyal** [comment]
Thanks for your interest in our work. 

Yes, for each experiment, we do multiple runs with different seeds. As mentioned in the paper, we do 4 runs and report the mean and variance. We recommend doing so for reliable performance. 

No extra preprocessing needs to be done. 

For debugging purposes, please share the commands you used for training and testing. Also, please let us know if any other changes were done.
